src/components/data/FileBrowser.js

```diff
@@ -424,6 +454,11 @@ const BucketBrowser = (({
         onSuccess: ({ name }) => {
           setCreatingNewFolder(false)
           setPrefix(`${prefix}${name}/`)
+
+          // Create a placeholder object for the new folder.
+          // See https://cloud.google.com/storage/docs/folders for more information.
+          const placeholderObject = new File([''], `${name}/`, { type: 'text/plain' })
+          Ajax().Buckets.upload(googleProject, bucketName, prefix, placeholderObject)
```

Comment:
What if this Ajax call were to throw an error?

Reply:
Added error reporting in https://github.com/DataBiosphere/terra-ui/pull/3246/commits/392c79c057aa7e03fbcbdca80723c8da2feff872.

Reply:
Why does withErrorReporting not work? Is it because you need to set the busy flag without using the spinner? What does the busy flag control?

Reply:
I guess it does. I kind of got in the habit of not using the wrappers after running into several "state change on unmounted component" warnings when using `Utils.withBusyState` in modals. Switched to using it in https://github.com/DataBiosphere/terra-ui/pull/3246/commits/c433d9d31ab25e2571dcfa2494a2af5100e548fb.

The busy flag controls the spinner.